intro-HPC/ch_running_jobs_with_input_output_data.tex

```diff
@@ -533,7 +533,19 @@ \subsection{Check your quota}
 status of your quota in a readable format:
 
 \begin{prompt}
-\shellcmd{show_quota}
+%\shellcmd{show\_quota}%
+VSC_DATA:    used 81MB (0%\%%)  quota 25600MB
+VSC_HOME:    used 33MB (1%\%%)  quota 3072MB
+VSC_SCRATCH:   used 28MB (0%\%%)  quota 25600MB
+VSC_SCRATCH_GLOBAL: used 28MB (0%\%%)  quota 25600MB
+VSC_SCRATCH_SITE:   used 28MB (0%\%%)  quota 25600MB
+\end{prompt}
```

Comment:
I think these should maybe be site-specific. I guess the above are for Leuven (which is not mentioned, but you do explicitly mention UA below. 

If you make it site-specific, UGent has e.g.,

~~~~{.bash}
User quota:
VSC_HOME: used 1.12 GiB (39%) quota 2.85 GiB (3 GiB hard limit)
VSC_DATA_VO: used 708 GiB (42%) quota 1.62 TiB (1.71 TiB hard limit)
VSC_DATA: used 8.95 GiB (37%) quota 23.8 GiB (25 GiB hard limit)
VSC_SCRATCH_DELCATTY_VO: used 514 GiB (11%) quota 4.52 TiB (4.76 TiB hard limit)
VSC_SCRATCH_DELCATTY: used 619 MiB (2%) quota 23.8 GiB (25 GiB hard limit)
VSC_SCRATCH_PHANPY_VO: used 0 B (0%) quota 486 MiB (512 MiB hard limit)

VO quota:
VSC_DATA_VO: used 2.05 TiB (63%) quota 3.25 TiB (3.42 TiB hard limit)
VSC_SCRATCH_DELCATTY_VO: used 4.02 TiB (44%) quota 9.05 TiB (9.52 TiB hard limit)
VSC_SCRATCH_PHANPY_VO: used 0 B (0%) quota 972 GiB (1 GiB hard limit)
~~~~

The command used is `show_quota --vo`. We also store this information in the account page, so if they visit that, UGent users also see the quota and usage information.